services/auth_service.py

```
@@ -0,0 +1,157 @@
+            data = json.loads(raw)
+        except json.JSONDecodeError as e:
+            raise AuthStoreError(
+                f"凭据文件已损坏（JSON 解析失败）{self.store_path}: {e}"
+            ) from e
+        if (
+            not isinstance(data, dict)
+            or not data.get("username")
+            or not data.get("password_hash")
+        ):
+            raise AuthStoreError(
+                f"凭据文件内容不完整（缺少 username/password_hash）{self.store_path}"
```

Ruff (0.15.17)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)